local/local_test.go

```diff
@@ -396,7 +395,7 @@ func testImage(t *testing.T, when spec.G, it spec.S) {
 		})
 
 		when("#WithConfig", func() {
-			var config = &container.Config{Entrypoint: []string{"some-entrypoint"}}
+			var config = &v1.Config{Entrypoint: []string{"some-entrypoint"}}
```

Comment:
This lifecycle (which uses `WithConfig`) will need to be updated before we can update imgutil